docs/design/hostNetwork.md

```diff
@@ -0,0 +1,47 @@
+# Enabling  host networking for controller plugin pods
+
+By default, the ceph-csi controller plugins operate on the pod network
```

Comment:
`ceph-csi` to `Ceph-CSI` lets use the same convention

Reply:
done